iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/Devices.java

```diff
@@ -12,38 +14,58 @@
 import com.microsoft.azure.sdk.iot.service.devicetwin.*;
 import com.microsoft.azure.sdk.iot.service.exceptions.IotHubException;
 import com.microsoft.azure.sdk.iot.service.exceptions.IotHubNotFoundException;
+import org.apache.commons.collections4.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
 import play.Logger;
 import play.libs.Json;
 
 import java.io.IOException;
 import java.util.*;
-import java.util.concurrent.*;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ExecutionException;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
 
 public final class Devices implements IDevices {
 
     private static final Logger.ALogger log = Logger.of(Devices.class);
 
     private static final int MAX_GET_LIST = 1000;
-    private static final String QueryPrefix = "SELECT * FROM devices";
+    private static final String DevicesQueryPrefix = "SELECT * FROM devices";
+    private static final String ModulesQueryPrefix = "SELECT * FROM devices.modules";
 
```

Comment:
Do you think we should have ENUMs or separate constant files?

Reply:
I don't think so. If we ended up with a few more queries but I don't see this growing to be more prefixes since this is everything the hub supports.

Reply:
sure, just wanted to confirm if we have anything more coming up. thanks